src/main/java/core/basesyntax/service/implementions/RecordDtoParserImpl.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.service.implementions;
+
+import core.basesyntax.dto.FruitRecordDto;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.OperationType;
+import core.basesyntax.service.RecordDtoParser;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RecordDtoParserImpl implements RecordDtoParser {
+    private static final String COMMA = ",";
```

Comment:
```suggestion
    private static final String CSV_SEPARATOR = ",";
```
we should name variable according to their purpose/meaning instead of value that it holds

Reply:
done